### DISCUSSION TITLE
improve screensaver and play multiple videos

### DISCUSSION THREAD

**Discussion by smoothnobody** [rocksdanister/lively#1879] [Ideas]
check out this screensaver.
https://sourceforge.net/projects/videosaver/

it can select multiple video files and play them at random.

you should incorporate something like this in to lively. 

where it needs improvement, it doesn't support many video formats, it doesn't support H265 or HDR.

when a video ends, it won't start a new video, it just loops the same video until the program starts again.

lively is cool, but your SCR screensaver is bare bones. the biggest problem it only plays the current lively wallpaper. why would i want the wallpaper and screensaver to be the same thing? seeing the same thing all the time gets boring. it would be nice if both the screensaver and the wallpapers could play randomized videos. you should have a folder and it plays every file in the folder at random. when the video ends it should play another file.

**Comment by rocksdanister** [comment]
dup: https://github.com/rocksdanister/lively/issues/587
dup: https://github.com/rocksdanister/lively/issues/137

**Comment by smoothnobody** [comment]
you have been saying it's planned and under development since 2020.

**Reply by rocksdanister** [reply]
You creating duplicate threads for it is not helping anyone :)

If you want the feature feel free to send a pull request.

Lively is just a personal project I do for fun.. no feature will have a fixed timeline.
